drydock_backups/plugin.py

```diff
@@ -15,7 +15,7 @@
         "VERSION": __version__,
         "IMAGE": "ednxops/shipyard-utils:v1.1.0",
         "CRON_SCHEDULE": '0 2 * * *',
-        "BACKUP_STORAGE_SERVICE": "aws-s3",
+        "STORAGE_SERVICE": "aws-s3",
```

Comment:
is the name change correct?

Reply:
yes, if you prepend the string `BACKUP_` to all variables in the 'default' dictionary [here](https://github.com/eduNEXT/drydock/blob/mar/fix-drydock-backups/drydock_backups/plugin.py#L37-L43).

Reply:
@marbonilla should you mark this comment as resolved?